packages/caip/src/adapters/gem/index.ts

```diff
@@ -0,0 +1,35 @@
+import toLower from 'lodash/toLower'
+
+const CAIP19ToGemTickerMap = {
```

Comment:
They are... kind of.
There is no such information provided by the API and one ticker could mean native for one asset, ERC20 for another. Documentation of coinify and wyre is the source of truth here, without something like CAIP provided by these two services. 

I manually checked all the assets we get from Gem API against Wyre and Coinify docs to identify which network is used for a specific asset, to make sure it is the version we use (ERC20) - ditched the ones we do not support either because they're native ones that we don't support, or because we simply don't have support for that ERC20 in the app. 

There are also some tickers like "BNBBSC" (BNB on BSC vs. ERC20 for the "BNB" ticker) that tie an asset to its chain-specific token - that we could use in the future.

Reply:
@gomesalexandre can you link to the api you're hitting to get these tickers? i feel like when we discussed this they looked different and a bit more specific

Reply:
it might seem anal, but i'd strongly prefer if we used the `gem_asset_id` over the `ticker`

i'm more confident the `gem_asset_id` is unlikely to change, vs the `ticker`

Reply:
i did check, and the tickers are unique, for now, but i'm not confident they won't have duplicates in future. it's kind of like relying on symbol on an erc20.